src/pages/MainPage/StarRate.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
```

Comment:
remove this and put a place holder for now
we will fix it later but make sure to add this issue to the backLog in our project